spec/features/users/regular_users/edit_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe "User Profile" do
+  before :each do
+    @regular_user = User.create!(name: "George Jungle",
+                  address: "1 Jungle Way",
+                  city: "Jungleopolis",
+                  state: "FL",
+                  zipcode: "77652",
+                  email: "junglegeorge@email.com",
+                  password: "Tree123")
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@regular_user)
```

Comment:
In my limited understanding of stubs, I don't understand why this stub is necessary, so it would be great if you can explain this to me. :)